src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -35,13 +35,14 @@
 session.mount("http://", adapter)
 session.mount("https://", adapter)
 
+API_URL = settings.MAIL_PROVISIONING_API_URL
+API_CREDENTIALS = settings.MAIL_PROVISIONING_API_CREDENTIALS
+API_TIMEOUT = settings.MAIL_PROVISIONING_API_TIMEOUT
+
 
 class DimailAPIClient:
     """A dimail-API client."""
 
-    API_URL = settings.MAIL_PROVISIONING_API_URL
```

Comment:
I prefer when settings comes from the class, it makes changes for test or script easier ^^